File: src/js/pe_0005.js
var n = 0;
var check = 0;

while ( check == 0 ) {
    check = 1;
    n += 20 * 19;
    for (var m = 19; 1 <= m; --m) {
        if ( n % m != 0 ) {
            check = 0;
            break;
        }
    }
}

console.log(n);

File: src/jl/pe_0005.jl
function main()
    n = 0
    check = 0

    while (check == 0)
        check = 1
        n += 20 * 19
        for m in 19:-1:1
            if (n % m != 0)
                check = 0
                break
            end
        end
    end

    println(n)
end

main()
